packages/grid/test/accessibility.common.js

```diff
@@ -359,4 +359,33 @@ describe('accessibility', () => {
       });
     });
   });
+
+  describe('accessibleName', () => {
```

Comment:
This suite is actually missing its setup. If you run this suite alone, the tests fail because `grid` is undefined.

Reply:
Added the test suite setup.